pf/tests/internal/testprovider/tls.go

```diff
@@ -54,13 +54,14 @@ func TLSProvider() tfbridge.ProviderInfo {
 	}
 
 	return tfbridge.ProviderInfo{
-		Name:        "tls",
-		P:           tfpf.ShimProvider(tlsshim.NewProvider()),
-		Description: "A Pulumi package to create TLS resources in Pulumi programs.",
-		Keywords:    []string{"pulumi", "tls"},
-		License:     "Apache-2.0",
-		Homepage:    "https://pulumi.io",
-		Repository:  "https://github.com/pulumi/pulumi-tls",
+		Name:             "tls",
+		P:                tfpf.ShimProvider(tlsshim.NewProvider()),
+		Description:      "A Pulumi package to create TLS resources in Pulumi programs.",
+		Keywords:         []string{"pulumi", "tls"},
+		License:          "Apache-2.0",
+		Homepage:         "https://pulumi.io",
+		Repository:       "https://github.com/pulumi/pulumi-tls",
+		UpstreamRepoPath: ".",
```

Comment:
Is this a new required field that all bridged providers will need to add?

Reply:
No. It is an existing field, and it is not required. I wanted some of our test providers to have their docs lookup path in valid go repos, while unaltered providers have their docs lookup path in invalid go repos.

Existing providers will not need to take any action after this PR merges.